docs/core/linux-prerequisites.md

```diff
@@ -122,6 +122,15 @@ For versions earlier than .NET Core 2.1, following dependencies are also require
 * libunwind8
 * libuuid1
 
+For .NET Core applications which use the System.Drawing.Common assembly, you will also need the following dependencies:
+
+* libgdiplus (version 6.0.1 or later)
```

Comment:
also wondering if we shouldn't just append this to the first list like this instead of creating a third list of dependencies

```suggestion
* libgdiplus (version 6.0.1 or later) - required by applications that use the *System.Drawing.Common* assembly.
```